vuepress/docs/next/docs/getting-started/development-concepts.md

```diff
@@ -11,7 +11,7 @@ Entando provides a distributed and composable environment in which components ca
 ![Entando Platform Development Process](./img/concepts.png)
 
 ## Create Components 
-The first step in building a modern application on Entando is to design and create each independent component or functional block. Developers can use the [Entando Component Generator](../create/component-gen-overview.md) and [Blueprint](../create/blueprint-features.md) to automate the building and packaging of project files, with the flexibility to customize. The modular design of Entando applications means new business capabilities can be integrated into your application without reworking existing systems. 
+The first step in building a modern application on Entando is to design and create each independent component or functional block. Developers can use the [Entando Component Generator](../create/component-gen-overview.md) and [Blueprint](../create/blueprint-features.md) to automate the building and packaging of project files, with the flexibility to customize. The modular design of apps built on Entando means new business capabilities can be integrated into your application without reworking existing systems. 
```

Comment:
one last play...

can we rephrase to "Apps built on Entando leverage a modular design to integrate new business capabilities without reworking existing systems" ?? same kind of thing for other sentences with "means"??

if "no" i'll forever hold my peace...